packages/manager/modules/billing/src/billing.routing.js

```diff
@@ -25,6 +29,53 @@ export default /* @ngInject */ ($stateProvider) => {
       },
       goToOrders: /* @ngInject */ ($state) => () =>
         $state.go('app.account.billing.orders'),
+
+      guides: /* @ngInject */ (guidesObjectLevelOne) => {
+        return {
+          url: guidesObjectLevelOne(GUIDES_LIST),
+          tracking: GUIDE_TRACKING_TAG,
+        };
+      },
+
+      guidesObjectLevelOne: /* @ngInject */ (guidesObjectLevelTwo) => (
+        object,
+      ) => {
+        return Object.fromEntries(
+          Object.entries(object).map(([key, value]) => {
+            return [key, guidesObjectLevelTwo(value)];
+          }),
+        );
+      },
+
+      guidesObjectLevelTwo: /* @ngInject */ (guidesObjectLevelThree) => (
+        object,
+      ) => {
+        return Object.fromEntries(
+          Object.entries(object).map(([key, value]) => {
+            return [key, guidesObjectLevelThree(value)];
+          }),
+        );
+      },
```

Comment:
**suggestion:** These two functions looks same, we can have one function which accepts **object** and **callback** function.

Reply:
That's right, but I haven't found a way to reuse the same one.

Reply:
In addition to @anooparveti review, maybe we could use recursivity here :)